kernel/rtlil.cc

```diff
@@ -922,9 +922,9 @@ void RTLIL::Module::makeblackbox()
 	set_bool_attribute(ID::blackbox);
 }
 
-void RTLIL::Module::reprocess_module(RTLIL::Design *, const dict<RTLIL::IdString, RTLIL::Module *> &)
+void RTLIL::Module::expand_interfaces(RTLIL::Design *, const dict<RTLIL::IdString, RTLIL::Module *> &)
```

Comment:
Given we're adjusting all of the existing `reprocess_module` stuff, I think we should just use `dynamic_cast` at the call site and avoid this virtual-but-not-really shenanigans.

Reply:
I agree, but I did this to match the code for `derive`. Are you happy if we tidy this up as a follow-on?